<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Station</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qtResources.qrc">
    <normaloff>:/newPrefix/ui/marta-logo.png</normaloff>:/newPrefix/ui/marta-logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {background-color: #ffffff} 
QLabel {color: #009fd7; font-weight: bold;}
#frameLabelTop {background-color: transparent; color: #ee9306; font-size: 30px;}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="stationNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Station Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="stopIdLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="entryFareLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entry Fare</string>
   </property>
  </widget>
  <widget class="QLabel" name="stationTypeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Station Type</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="busStationButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bus Station</string>
   </property>
  </widget>
  <widget class="QLabel" name="nearestIntersectionLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nearest Intersection</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="trainStationButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Train Station</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="openStationBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Station</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>When Checked, Passengers can enter at this station</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="createStationButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Station</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="windowModality">
    <enum>Qt::ApplicationModal</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">Line {backgournd-color: none;}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="frameLabelTop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {background-color: transparent}</string>
   </property>
   <property name="text">
    <string>Create New Station</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stationNameTextEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Station Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stopIdTextEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>89</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Stop Id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nearestIntersectionTextEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Nearest Intersection</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="entryFareSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="prefix">
    <string>$</string>
   </property>
   <property name="maximum">
    <double>50.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="returnStationButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Station Management</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../qtResources.qrc"/>
 </resources>
 <connections/>
</ui>
